README.rst

```diff
@@ -60,15 +60,15 @@ Requirements
 
  * `pass <http://www.passwordstore.org/>`_
  * `xclip <http://sourceforge.net/projects/xclip>`_
- * `python <https://www.python.org/>`_
+ * `python 3 <https://www.python.org/>`_
```

Comment:
@hobarrera 
Fixed README.
I don't know what is the minimum version, but I assumed that at least 3.3 is required.